bot/exts/utils/utils.py

```diff
@@ -166,6 +168,21 @@ async def zen(self, ctx: Context, *, search_value: Union[int, str, None] = None)
         embed.description = best_match
         await ctx.send(embed=embed)
 
+    @command(aliases=("snf", "snfl", "sf"))
+    @in_whitelist(channels=(Channels.bot_commands,), roles=STAFF_ROLES)
+    async def snowflake(self, ctx: Context, snowflake: int) -> None:
```

Comment:
Seems like the Snowflake converter could fit here

Reply:
What? Custom converter?

Reply:
Solved in 47a2607